grafana/grafana-amazonprometheus-datasource
# Prometheus backend with promlib

- Use `promlib` https://github.com/grafana/grafana/releases/tag/pkg%2Fpromlib%2Fv0.0.2
- We still need this one to be merged https://github.com/grafana/grafana/pull/84462
- Use middlewares 
  - ResponseLimitMiddleware
  - SigV4Middleware
  - _I will check if we need more_ https://docs.google.com/document/d/1CgjbHttaePX2el4fUEIPmtTuJGoj0d-wpoB3LCcqX3A/edit#heading=h.5sybau7waq2q

### How to test
In `prometheus-amazon`, run:
- `yarn dev` -> Frontend
- `mage -v` -> Backend

In grafana/grafana add the following in `conf/custom.ini` file

```
[paths]
plugins=<directory-that-you-have-prometheus-amazon-repo>
```

Run grafana as usual. You should be able to create a `prometheus-amazon` datasource
